Loop/Managers/LoopAppManager.swift

```diff
@@ -290,6 +290,8 @@ class LoopAppManager: NSObject {
 
         loopDataManager = LoopDataManager(
             lastLoopCompleted: ExtensionDataManager.context?.lastLoopCompleted,
+            publishedMostRecentGlucoseDataDate: ExtensionDataManager.context?.mostRecentGlucoseDataDate,
```

Comment:
This has always been a little odd that Loop's storage for `lastLoopCompleted` was in the status extension. Now that the status extension is dropped, these things should probably move to just user defaults. I didn't notice that ExtensionDataManager was not removed in Nate's PR.  I don't think it's used anymore except maybe for this.